.github/workflows/pyopenms-wheels.yml

```
@@ -127,9 +127,17 @@ jobs:
-          ${CURRENT_PYTHON_EXECUTABLE} -m pip install -r $GITHUB_WORKSPACE/OpenMS/src/pyOpenMS/requirements_bld.txt
+          # Install uv and tomli for faster dependency installation
+          ${CURRENT_PYTHON_EXECUTABLE} -m pip install --upgrade pip uv tomli
+          # Install build dependencies from pyproject.toml [dependency-groups].build
+          uv pip install $(python -c "
+try:
+    import tomllib
+except ImportError:
+    import tomli as tomllib
+with open('$GITHUB_WORKSPACE/OpenMS/src/pyOpenMS/pyproject.toml', 'rb') as f:
+    print(' '.join(tomllib.load(f)['dependency-groups']['build']))
+")
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)